README.md

```diff
@@ -4,19 +4,45 @@
 
 Data for the dashboard is going to be provided by a backend in the future, but such an API is not in place yet (imagine your colleagues are working on it). Provided already is an exemplary JSON file that resembles a response from the planned API.
 
-Use a non-proprietary tech stack of your choice and explain in a readme how to run your solution. Please email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). If you're using
-Git during building your application please make sure to include the Git history as well.
+Use a non-proprietary tech stack of your choice and explain in a readme how to run your solution. Please use version control (git is preferred).
+
+## Submitting your solution
+
+Please submit your solution at least 2-3 days before your technical interview. You have two options:
+
+1. Hosted repository (preferred)
+
+Email a link to a GitHub/Bitbucket/GitLab/etc. repo to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to make the repo private 3 days after the interview.
+
+
+2. Email 
+
+You can also email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to include the commit history in your submission.
 
 ## Evaluation
 
-You should turn in a complete and runnable solution, but do not gold-plate it: we don't expect you to spend more than a few hours on this exercise. We want to see that you know how to use APIs efficiently, how you structure and how you are testing your application. It should be easy to tell from the dashboard which ministries have provided the most data.
+We want to use this challenge to understand how you would approach such a problem in a work setting. Here are some specifics of what we are looking for:
+
+Among other things, we pay attention to:
+- how you structure your application and your source code,
+- how you name variables/functions/classes/packages,
+- how you test your application and what kinds of tests you're writing,
+- how you document your application, including instructions to run the project,
+- what your commits look like,
+- whether you know how to consume APIs efficiently,
+- what tooling you use,
+- overall consistency & attention to detail.
+
+You should turn in a complete and runnable solution, but do not gold-plate it: we don't expect you to spend more than a few hours on this exercise.
 
 In the technical interview that follows we would like to discuss your approach, and that includes talking about areas you would want to improve if given more time.
 
+Note that the UI won't be part of the evaluation (could be plain HTML for instance).
```

Comment:
I think this was copied from the backend challenge? I think here we want to evaluate the UI? 😬

Reply:
+1 :)